encryption-service-vault/src/main/java/com/mx/path/facility/security/vault/VaultEncryptionServiceConfiguration.java

```diff
@@ -0,0 +1,81 @@
+package com.mx.path.facility.security.vault;
+
+import java.time.Duration;
+
+import lombok.Data;
+
+import com.mx.common.configuration.ConfigurationField;
+
+@Data
+public class VaultEncryptionServiceConfiguration {
+
+  private static final int DEFAULT_ENGINE_VERSION = 1;
+  private static final String DEFAULT_URI = "http://127.0.0.1:8200";
+  private static final int DEFAULT_MAX_RETRIES = 0;
+  private static final int DEFAULT_RETRY_INTERVAL_MILLISECONDS = 200;
+  private static final String DEFAULT_AUTHENTICATION = "APPID";
```

Comment:
Could change the default authentication to 'APPROLE' since we are completely on K8s now...

Reply:
Good point. @RyanJones08 Do you have an opinion?

Reply:
Yes we can change to the default being `APPROLE` now that we are fully on k8s and remove the deprecated `APPID` since nothing is using that anymore.